docker-compose/docker-compose.sample.yml

```diff
@@ -1,26 +1,37 @@
 version: "3.7"
 
 services:
-  phpfpm:
-    image: modestcoders/php:7.1-fpm
+
+  nginx:
```

Comment:
I don't thing we really need to flip phpfom and nginx. Is there any idea behind it?